modules/openapi-generator/src/test/resources/3_0/python/petstore-with-fake-endpoints-models-for-testing.yaml

```diff
@@ -1957,6 +1957,13 @@ components:
           properties:
             breed:
               type: string
+    Hunting__Dog:
+      allOf:
+        - $ref: '#/components/schemas/Animal'
```

Comment:
thanks for adding the test

to fix the build failure, what about using a new schema called "Creature" (based on Animal) to replace Animal in this new test schema instead? 

reusing Animal as parent causing build failure due to change in error message.

Reply:
@wing328 , sure 'Creature' also already exists, but I tried using that as the parent. If the build still fails I can create a whole new schema

Reply:
all good now

thanks for adding the test